connector/tracetotopoconnector/spec/settings.yaml

```diff
@@ -19,18 +19,64 @@ externalDocs:
 paths: {}
 components:
   schemas:
-    SettingsSnapshot:
+    SettingsProtocol:
```

Comment:
I think it would make more sense to move the spec and generated code to the `settings_extension` module or maybe in its own module. Now we get a bit of a weird (almost?) circular dependency between the settings extension and the connector.